demos/smart_classroom_demo/src/action_detector.cpp

```diff
@@ -108,6 +107,11 @@ ActionDetection::ActionDetection(const ActionDetectorConfig& config)
                 const auto anchor_dims = outputInfo[glob_anchor_name]->getDims();
                 anchor_height = new_network_ ? anchor_dims[2] : anchor_dims[1];
                 anchor_width = new_network_ ? anchor_dims[3] : anchor_dims[2];
+                std::size_t action_dimention_idx = new_network_ ? 1 : 3;
+                if (anchor_dims[action_dimention_idx] != config_.num_action_classes) {
+                    throw std::logic_error("The number of specified actions and the number of actions predicted by "
+                        "the Person/Action Detection Retail model must match");
```

Comment:
Retail?

Reply:
That is how the demo refers to the models